src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -118,12 +118,14 @@ public LaunchType SelectedLaunchType
                         SelectedDebugProfile.CommandName = _selectedLaunchType.CommandName;
                         if (_selectedLaunchType.CommandName == ProfileCommandNames.Executable)
                         {
-                            ExecutablePath = String.Empty;
-                        }
-                        else if (_selectedLaunchType.CommandName == ProfileCommandNames.IISExpress)
-                        {
-                            ExecutablePath = String.Empty;
-                            HasLaunchOption = true;
+                            if (SelectedDebugProfile != null && !String.IsNullOrWhiteSpace(SelectedDebugProfile.ExecutablePath))
+                            {
+                                ExecutablePath = SelectedDebugProfile.ExecutablePath;
```

Comment:
The setter for `ExecutablePath` will just store the value back to `SelectedDebugProfile.ExecutablePath`, so this line shouldn't be needed.  I think a simpler way to write the code would be:

```C#
if (ExecutablePath == null)
{
    ExecutablePath = String.Empty;
}
```